.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit:
```

Comment:
Maybe a more descriptive name?

```suggestion
  unit-tests:
```

Reply:
The naming conventions were with an eye towards how it looks in the PR checks:  
![Screen Shot 2020-10-24 at 2 34 14 PM](https://user-images.githubusercontent.com/4371429/97090942-05ed8880-1606-11eb-8bcb-84e229b5c0a9.jpg)

Actually kind of suggests the workflow should be "test" (all lowercase). Thoughts?

Reply:
"test" is even better.

Reply:
Ohh. Sorry. I just got what you meant.

For the workflow name, I suggest "CI". 
The name is delightfully generic, so we can add separate jobs for something like a linter in the future and the name still describes it all.

As for the name of this job: Maybe just "tests"?

Reply:
Ok, I think this is good.